opm/simulators/linalg/ISTLSolverEbosFlexible.hpp

```diff
@@ -136,13 +153,49 @@ class ISTLSolverEbosFlexible
             // Never recreate solver.
         }
 
+        std::function<VectorType()> weightsCalculator;
+
+        auto preconditionerType = prm_.get("preconditioner.type", "cpr");
+        if( preconditionerType  == "cpr" ||
```

Comment:
Nitpick (applies to the whole PR): there are lots of minor formatting issues here. I prefer that you clang-format them away (since they are in new code), but we can do it separately later.

Reply:
Someone else should to that later. Removing tabs from the original PR was already quite exausting...

And beware it might change lot of places not touched here and prevent future cherry-picks, reversions, etc.

Reply:
Ok, note that my suggestion is not to clang-format the files wholesale, but just the new parts (not sure what editor you are using, but I have set up emacs so that `M-x clang-format` modifies the current selection, very useful).